experiments/targeting.go

```diff
@@ -291,7 +292,27 @@ func (n *ComparisonNode) Evaluate(inputs map[string]interface{}) bool {
 	case int:
 		return n.comparer(float64(cv), value)
 	case float64:
-		n.comparer(cv, value)
+		return n.comparer(cv, value)
```

Comment:
Looks like this was broken for `float64` too since there was no `return` statement.

Reply:
Can you reorganize this so `float64` is after `float32` so it flows up naturally?